core/src/main/resources/lib/form/filter-menu-button/filter-menu-button.js

```diff
@@ -39,7 +39,7 @@ function _createFilterMenuButton(menu) {
   filterInput.setAttribute("type", "search");
 
   filterInput.addEventListener("input", (event) =>
-    _applyFilterKeyword(menu, event.currentTarget)
+    _applyFilterKeyword(menu, event.currentTarget),
```

Comment:
```suggestion
    _applyFilterKeyword(menu, event.currentTarget)
```

Reply:
This suggestion was incorrect. HtmlUnit was complaining about line 6, not line 43. See commit 8aee10ea8a1c03c71868bd116e08efa1c9bbc1af.